cfy_manager/components/composer/composer.py

```diff
@@ -93,13 +89,6 @@ def _verify_composer_alive(self):
         wait_for_port(COMPOSER_PORT)
 
     def _start_and_validate_composer(self):
-        # Used in the service template
-        config[COMPOSER][SERVICE_USER] = COMPOSER_USER
-        config[COMPOSER][SERVICE_GROUP] = COMPOSER_GROUP
-        systemd.configure(COMPOSER,
-                          user=COMPOSER_USER, group=COMPOSER_GROUP)
-
-        logger.info('Starting Composer service...')
         systemd.restart(COMPOSER)
```

Comment:
Could these two lines just be under start()? Then _start_and_validate can be scrapped.